README.md

```diff
@@ -190,17 +199,17 @@ supported value is `application/json`, the default.
 the provided (presumably user-written) Flattener.
 
 `WithPatternDeletion`: If true, arranges that Quamina
-allows Patterns to be deleted from an instance. This is 
-not free; it can incur extra costs in memory and 
+allows Patterns to be deleted from an instance. This is
+not free; it can incur extra costs in memory and
 occasional stop-the-world Quamina rebuilds. (We plan
 to improve this.)
 
 `WithPatternStorage`: If you provide an argument that
 supports the `LivePatternStorage` API, Quamina will
-use it to 
+use it to
```

Comment:
nit: weird linewrapping

Reply:
Fixed

Reply:
Haha, as an old XML guy having weird line-breaks in the source doesn't bother me, as long as the formatted .md 
looks OK.  But I can see your point so I fixed these.